app/models/release.rb

```diff
@@ -56,9 +56,15 @@ def assign_release_number
     # release-number-from-ci plugin.
     return if number != DEFAULT_RELEASE_NUMBER && number.present?
 
-    latest_release_number = project.releases.last.try(:number) || "0"
+    raise "Unable to auto bump version" unless self.number = next_release_number
```

Comment:
Something is funny here.  I'm not sure if it needs an `attr_writer`, but if I remove the `self`, it doesn't stick.